components/omega/test/testdummy.cpp

```diff
@@ -1,3 +1,4 @@
+#include "logging.h"
```

Comment:
Shouldn't there be a `testlogging.cpp`, too?  Something that has unit tests for each of the macros?

Reply:
You're right. I did unit tests but not "officially" through CMake. I am writing a draft code for the framework of Omega uni tests, and will add logging unit test as the first one of the framework.

Reply:
See the DataType and MachEnv unit tests for examples of a unit test driver and place this one in test/base/LoggingTest.cpp. Output should be include the PASS or FAIL words for each test so that CTest can grep.

Reply:
I am rewriting logging test according to DataTypesTest.cpp. In addition, I am adding more logging tests that are missed in this PR.